scripts/lib/CIME/case.py

```diff
@@ -1063,6 +1069,9 @@ def create_caseroot(self, clone=False):
                       "README.case", caseroot=self._caseroot)
         for component_class in self._component_classes:
             if component_class == "CPL":
+                append_status("Using %s coupler instances" %
+                              (self.get_value("COUPLER_COUNT")),
+                              "README.case", caseroot=self._caseroot)
```

Comment:
In addition to the number of couplers, please add the number of instances to README.case